src/sass/components/mainmenu.scss

```diff
@@ -1,3 +1,9 @@
+.mainmenu__container {
```

Comment:
You could make these custom components individual folders like components in the uikit. It would make it easier to contribute in the future.

Reply:
Not sure what you mean here. This SASS is pretty much the same as Design System where I copied it from

Reply:
Yeah I just meant we can make a folder with a component e.g.
```
main-menu
- module.scss
- print.scss
- global.scss
```

Reply:
Yep that's nice and clean. Our project just has one SASS per component.